image_generation/lcm_dreamshaper_v7/cpp/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(CMAKE_BUILD_TYPE "Release" CACHE STRING "CMake build type")
 
 # dependencies
 
-find_package(OpenVINO REQUIRED COMPONENTS Runtime)
+find_package(OpenVINO REQUIRED COMPONENTS Runtime Threading)
```

Comment:
FYI: will be reverted once https://github.com/openvinotoolkit/openvino/pull/26269 is merged and used within GenAI